CMakeLists_files.cmake

```diff
@@ -58,7 +58,10 @@ if(ENABLE_ECL_INPUT)
     src/opm/parser/eclipse/EclipseState/AquiferConfig.cpp
     src/opm/parser/eclipse/EclipseState/AquiferCT.cpp
     src/opm/parser/eclipse/EclipseState/Aquifetp.cpp
+    src/opm/parser/eclipse/EclipseState/NumericalAquifer.cpp
```

Comment:
OK - when you are in this part of town. Can you make a `Aquifer/` directory and put all the aquifer stuff there.

Reply:
Will do it.